apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template-inserter.php

```diff
@@ -124,6 +127,35 @@ private function fetch_retry( $request_url, $request_args = null, $attempt = 1 )
 		$this->fetch_retry( $request_url, $request_args, $attempt );
 	}
 
+	/**
+	 * Returns a default header if call to template api fails for some reason.
+	 *
+	 * @return string Content of a default header
+	 */
+	public function get_default_header() {
+		return '<!-- wp:a8c/site-description /-->
+			<!-- wp:a8c/site-title /-->
+			<!-- wp:a8c/navigation-menu /-->
+			<!-- wp:paragraph -->
+				<p>' .
+				__(
+					'The theme did not activate correctly so it may not look identical to the demo site. 
+					You are still able to edit the header and footer content to suit your needs.',
+					'full-site-editing'
```

Comment:
Let's drop the warning message. Folks will end up having a published site with this visible, which isn't super helpful.

If we'd like to notify users, would recommend a plain notice component.

Reply:
I have dropped the warning message. I can't see an easy way to get a notification from this point in the process to a notice component on the front-end without setting a cookie, or saving some info to the database that can be picked up when the editor loads - I suggest we circle back on this if the trial indicates that a lot of users are ending up in this state.